### DISCUSSION TITLE
FullCalendar Element

### DISCUSSION THREAD

**Discussion by frankvp11** [zauberzeug/nicegui#2048] [Ideas / Feature Requests]
In the discord we've been working on a FullCalendar element (https://fullcalendar.io/). I have a fork of the repo and it's almost ready to start a pull request, however for I need to figure out a way to import the FullCalendar library properly. Like, when I run my version of the documentation in a regular function it works fine, but in the main.py file it for some reason breaks. 
The relevant code can be found in my fork (https://github.com/frankvp11/nicegui), in the website/more_documentation/fullcalendar_documentation.py file. Any help would be greatly appreciated.

This is the error that I am currently dealing with:
![image](https://github.com/zauberzeug/nicegui/assets/90110502/c8ff614e-dd3d-4331-98ac-6abaa2346aac)

**Comment by frankvp11** [comment]
The short term fix until now has been to just add this 
```
    ui.add_head_html("<script src='https://cdn.jsdelivr.net/npm/fullcalendar@6.1.9/index.global.min.js'></script>")
```
however that doesn't appear to be working now - and I'd need to figure out how to have that script to be "used"

**Reply by falkoschindler** [reply]
Thanks for your effort, @frankvp11!
I think it would be easier for us if you'd just create the pull request, even if it's not finished yet. This way we can simply switch branches, add comments right in the code, track the development status etc. 🙂
